example/services/nginx/install_datadog.sh

```diff
@@ -1,61 +1,97 @@
-#!/bin/sh
+#!/usr/bin/env sh
+# Setup image with nginx if not installed and the corresponding version of
+# the nginx-datadog module.
 
 set -x
 set -e
 
+is_installed() {
+  if command -v "$1" > /dev/null 2>&1; then
+    return 1
+  else
+    return
+  fi
+}
+
+get_latest_release() {
+  curl --silent "https://api.github.com/repos/$1/releases/latest" | jq --raw-output .tag_name
+}
+
 if [ "$BASE_IMAGE" = '' ]; then
   >&2 echo 'This script expects BASE_IMAGE to be in the environment, e.g. "nginx:1.23.1-alpine" or "amazonlinux:2.0.20220121.0".'
   exit 1
 fi
 
 case "$BASE_IMAGE" in
-    amazonlinux:*) nginx_modules_path=/usr/share/nginx/modules ;;
-    *) nginx_modules_path=/usr/lib/nginx/modules ;;
+    amazonlinux:*)
+      nginx_modules_path=/usr/share/nginx/modules
+      ;;
+    nginx:*)
+      nginx_modules_path=/usr/lib/nginx/modules
+      ;;
+    *)
+      >&2 echo 'Unsupported platform.'
```

Comment:
Oh, now I see. This is a little formatting and an added explicit "else" case, which previously was implicitly a no-op.

If docker compose is told to build an image, then failure to build is an error. If it's not told to build an image, than it will use an existing image if one exists, otherwise it's an error.

Adding this check is a good idea -- it probably makes for easier debugging if someone is trying different values for [BASE_IMAGE][1]. Consider this wording:
```suggestion
      >&2 echo 'BASE_IMAGE value "%s" is invalid. See nginx-version-info.example for more information.' "$BASE_IMAGE"
```

[1]: https://github.com/DataDog/nginx-datadog/blob/582d1cc992050246dc93c3e6fe80edafa8b86a1c/nginx-version-info.example#L20

Reply:
ah, my mistake, not yours :)

Please see my more recent comment.